ui/raidboss/data/02-arr/raid/t7.js

```diff
@@ -82,12 +82,12 @@
     },
     {
       id: 'T7 Voice',
-      regex: Regexes.gainsEffect({ effect: 'Cursed Voice' }),
-      regexDe: Regexes.gainsEffect({ effect: 'Stimme Der Verwünschung' }),
-      regexFr: Regexes.gainsEffect({ effect: 'Voix Du Maléfice' }),
-      regexJa: Regexes.gainsEffect({ effect: '呪詛の声' }),
-      regexCn: Regexes.gainsEffect({ effect: '诅咒之声' }),
-      regexKo: Regexes.gainsEffect({ effect: '저주의 목소리' }),
+      netRegex: NetRegexes.gainsEffect({ effectId: '1C3', source: 'Melusine' }),
```

Comment:
Why are you adding `source` here? From your log line pictures that you linked to me, it doesn't look like there's a source.

Reply:
Ah, https://github.com/quisquous/cactbot/blob/master/docs/LogGuide.md#1a-networkbuff
`1A:[ObjectId]:[Target Name] gains the effect of [Status] from [Source Name] for [Float_Value] Seconds`.

I believed in an oversight on my part, if it is useless, I correct that.

Reply:
It's not that it's useless--it's that it's incorrect.  Looking at the log lines you sent me, there is no `source` from those gains effect lines.  It is empty.

Reply:
Yes, i understand. It's corrected.